tests/phpunit/integration/Core/Conversion_Tracking/Conversion_Event_Providers/WooCommerceTest.php

```diff
@@ -255,4 +256,174 @@ public function wgai_data_settings() {
 			),
 		);
 	}
+
+	/**
+	 * @dataProvider phone_normalization_data_provider
+	 * @runInSeparateProcess
+	 * @preserveGlobalState disabled
```

Comment:
Are these annotations mandatory? I tried removing them, and my local tests still passed.

Reply:
Removed `@preserveGlobalState disabled`.